pkg/ebpf/c/common/context.h

```diff
@@ -129,6 +130,23 @@ statfunc task_info_t *init_task_info(u32 tid, u32 scratch_idx)
     return bpf_map_lookup_elem(&task_info_map, &tid);
 }
 
+statfunc int init_policies_config(program_data_t *p)
+{
+    u16 version = p->config->policies_version;
+
+    // copy policies_config to event data
+    long ret = bpf_probe_read(
+        &p->event->policies_config, sizeof(policies_config_t), &p->config->policies_config);
+    if (unlikely(ret != 0))
+        return 1;
+
+    // finally set the version and set recompute_scope to true
+    p->event->context.policies_version = version;
```

Comment:
No need for the version variable since it is only used here
```suggestion
    p->event->context.policies_version = p->config->policies_version;
```